spec/features/general_feedback_spec.rb

```diff
@@ -6,13 +6,55 @@
     expect(page).to have_content(I18n.t('feedback.heading'))
   end
 
-  scenario 'successfully submitting feedback for the service' do
-    choose 'Find a job in teaching'
+  context 'when submitting feedback on the service' do
+    scenario 'must have a visit purpose' do
+      fill_in 'general_feedback_comment', with: 'Keep going!'
+      choose('general_feedback_user_participation_response_not_interested')
 
-    fill_in 'general_feedback_comment', with: 'Keep going!'
+      click_on 'Submit feedback'
 
-    click_on 'Submit feedback'
+      expect(page).to have_content('Visit purpose can\'t be blank')
+    end
 
-    expect(page).to have_content('Your feedback has been successfully submitted')
+    scenario 'must have a participation response' do
+      choose 'Find a job in teaching'
+      fill_in 'general_feedback_comment', with: 'Keep going!'
+
+      click_on 'Submit feedback'
+
+      expect(page).to have_content('User participation response can\'t be blank')
+    end
+
+    scenario 'successfully submitting feedback for the service' do
+      choose 'Find a job in teaching'
+      fill_in 'general_feedback_comment', with: 'Keep going!'
+      choose('general_feedback_user_participation_response_not_interested')
+
+      click_on 'Submit feedback'
+
+      expect(page).to have_content('Your feedback has been successfully submitted')
+    end
+
+    scenario 'must have an email when participation response is Yes' do
+      choose 'Find a job in teaching'
+      fill_in 'general_feedback_comment', with: 'Keep going!'
+      choose('general_feedback_user_participation_response_interested')
```

Comment:
You could extract this to a method for added clarity in the spec, perhaps something like `choose_yes`

Reply:
similarly for choose no